src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/_shipments.html.twig

```diff
@@ -1,9 +1,15 @@
 {% for shipment in order.shipments %}
+    {% set state =  shipment.state %}
     <div class="ui large icon message">
         <i class="truck icon"></i>
         <div class="content">
             <div class="header" id="sylius-shipping-method">
                 {{ shipment.method }}
+                <p id="shipping_status">
+                    {% if  shipment.state != 'cart'%}
```

Comment:
Doubled space after `if`. We could also move this if above `<p id="shipping_status">`